### ISSUE TITLE
Immersive mode on Android webview

### ISSUE THREAD

**Issue by subsoap** [defold/extension-webview#11]
If the project has immersive mode enabled the webview will not be - the bottom UI (with back button) will be visible.

**Comment by subsoap** [comment]
```
                        //| View.SYSTEM_UI_FLAG_LAYOUT_HIDE_NAVIGATION
                        | View.SYSTEM_UI_FLAG_LAYOUT_FULLSCREEN
                        //| View.SYSTEM_UI_FLAG_HIDE_NAVIGATION // hide nav bar
                        | View.SYSTEM_UI_FLAG_FULLSCREEN // hide status bar
                        | View.SYSTEM_UI_FLAG_IMMERSIVE
```

Uncommenting these helped to hide the bottom nav on webview though they might be there for a reason?

**Comment by britzl** [comment]
I'm not sure what kind of impact this might have. Ideally some of these View flags should be configurable. Have you noticed any difference?

**Comment by subsoap** [comment]
Not in functionality on the page, but it might be necessary for some features I'm not using.

**Comment by britzl** [comment]
Yeah, I think we need to make these into options when you create the webview.

**Comment by britzl** [comment]
I've played around with this a bit. The annoying thing is that the navigation bar becomes visible briefly when hiding/showing a webview. Did you experience this as well @subsoap ?

**Comment by subsoap** [comment]
@britzl Yes, I have seen this when opening / closing webviews (and it's annoying).

**Comment by baochungit** [comment]
My guess is it dues to the new window focused.
wm.addView(); --> new window/view focused
wm.removeView(); --> old window/view focused
I don't see it on my real device which running Android 12
